btn-1cc-checkout.js

```diff
@@ -0,0 +1,243 @@
+window.addEventListener('DOMContentLoaded', function() {
+
+  var btn = document.getElementById('btn-1cc');
+  var btnMini = document.getElementById('btn-1cc-mini-cart');
+  var btnPdp = document.getElementById('btn-1cc-pdp');
+  var rzpSpinnerBackdrop = document.getElementById('rzp-spinner-backdrop');
+  var rzpSpinner = document.getElementById('rzp-spinner');
+  var pageURL = jQuery(location).attr('href');
+  var url = new URL(pageURL);
+  var accessToken = new URLSearchParams(url.search).get('wcf_ac_token');
+
+  // event triggered by wc on any cart change
+  // as input function is the same, duplicate event listeners are NOT called
+  jQuery(document.body).on('updated_cart_totals', function(event) {
+    var btn = document.getElementById('btn-1cc');
+    if (btn !== null) {
+      btn.addEventListener('click', openRzpCheckout);
+    }
+
+    var btnMini = document.getElementById('btn-1cc-mini-cart');
+    if (btnMini !== null) {
+      btnMini.addEventListener('click', openRzpCheckout);
+    }
+  });
+
+  jQuery(document.body).on('wc_fragments_refreshed', function(event) {
+    var btnMini = document.getElementById('btn-1cc-mini-cart');
```

Comment:
Can this be made into a common function since it used in many places
`var btnMini = document.getElementById('btn-1cc-mini-cart');
    if (btnMini !== null) {
      btnMini.addEventListener('click', openRzpCheckout);
    }`